README.md

```diff
@@ -155,7 +155,7 @@ The following are the parameters for creating the cloudformation template:
 
 1. Clone the Trapheus Git repository
 2. From the Trapheus repo execute:  
-`sam package --template-file template.yaml --output-template-file deploy.yaml --s3-bucket <s3 bucket name from corresponding AWS account and region>`
+`sam package --template-file template.yaml --output-template-file deploy.yaml --s3-bucket <s3 bucket name from corresponding AWS account and region> --region <region to deploy Trapheus on for e.g. us-west-2`
```

Comment:
@stationeros i dont think the region parameter is required for compiling the changes using `sam package`. they are needed for the deployment of the stack as given below. Also, now i think `sam deploy` implicitly performs a `sam package` operation as well. could you re-confirm if this parameter is needed?

Reply:
@namitad What version of SAM are you using. Attaching the error. IMHO the documentation should be minimalistic but at the same time should not be prone to errors.

```
  File "/usr/local/Cellar/aws-sam-cli/1.23.0/libexec/lib/python3.8/site-packages/botocore/regions.py", line 148, in _endpoint_for_partition
    raise NoRegionError()
botocore.exceptions.NoRegionError: You must specify a region.
```

Also we must see what minimal version of sam is required so that package comes with deploy . Will check that and PR it separately once I have the correct information but as long as a command is there in readme I believe it should be correct. What do you think?